### PR TITLE
Add tests case for mariadb

### PR DESCRIPTION
**Pull request by LucasYao93**
Add tests case for mariadb.

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Comment by LucasYao93** [comment]
> Can one of the admins verify this patch?

Yes , I need assign to wyunchi-ms.

**Comment by LucasYao93** [comment]
Need assign to others user.

File: src/MariaDB/custom/Restore-AzMariaDBServer.ps1 [hunk 1]
```diff
@@ -179,7 +180,7 @@ function Restore-AzMariaDbServer
                 $Parameter.Location = $PSBoundParameters['Location']
                 $Null = $PSBoundParameters.Remove('Location')
             } else {
-                $Parameter.Location = $ServerObject.Location
+                $Parameter.Location = $Server.Location
```
Comment (wyunchi-ms) [inline src/MariaDB/custom/Restore-AzMariaDBServer.ps1:183]: Please revert this changes.

Comment (LucasYao93) [reply src/MariaDB/custom/Restore-AzMariaDBServer.ps1]: I have reverted this changes.
